src/components/Clamp.js

```diff
@@ -163,6 +163,7 @@ export default {
     },
     applyChange () {
       this.$refs.text.textContent = this.realText
+      this.$emit('change', this.isClamped)
```

Comment:
This should be quite frequent and "applying" changes to DOM is mostly used to measure the metrics during calculation, and it would take a while before the process reaches a stable result. It seems that what you need is a `clampchange` event, not `textchange`. Emitting events here doesn't seem to be correct as it exposes changes caused by the internal calculation process.

Reply:
fair enough. 
I've made some tests. It looks like the most reasonable way is to watch for the `offset` change since it's a direct assignment. Watching `isClaped` doesn't seem to work as expected without waiting for the $nextTick.
What do you think? Is it ok to watch the `offset`? Or maybe you have another suggestion.
Thanks!